docs/overview.adoc

```diff
@@ -953,13 +953,58 @@ Sometimes type information gets lost, these functions help you to coerce an "Any
 | apoc.node.degree.in(node, relationshipName) | returns total number of incoming relationship
 | apoc.node.degree.out(node, relationshipName) | returns total number of outgoing relationship
 | apoc.node.labels(node) | returns labels for (virtual) nodes
+| apoc.nodes.collapse([<node>], {config}) | returns the list of nodes merged into the first
 | apoc.rel.id(rel) | returns id for (virtual) relationships
 | apoc.rel.type(rel) | returns type for (virtual) relationships
 | apoc.any.properties(node/rel/map, [[keys]]) | returns properties for virtual and real, nodes, rels and maps, optionally restrict via keys
 | apoc.any.property(node/rel/map) | returns property for virtual and real, nodes, rels and maps
 | apoc.create.uuid() | returns a UUID string
 |===
 
+On `apoc.nodes.collapse` with config properties you can choose from 3 different behavior:
+ * "properties": "overwrite" : if there is the same property in more node, in the new one will have the last relationship's/node's property value
+ * "properties": "discard" : if there is the same property in more node, the new one will have the first relationship's/node's property value
+ * "properties": "combine" : if there is the same property in more node, the new one a value's array with all relationship's/node's values
+
+Relationships properties are managed with the same nodes' method, if properties parameter isn't set relationships properties are combined.
```

Comment:
combined should **not** be the default as it leads to gigantic array properties.